webui/Makefile

```diff
@@ -31,7 +34,7 @@ build-proto:
 
 .PHONY: build-api-ts-sdk
 build-api-ts-sdk: build-proto
-	$(MAKE) -C api-ts-sdk build
+	$(MAKE) -C api-ts-sdk get-deps build
```

Comment:
this part isn't parallelizable so probably we should have two lines instead of defining the dependency this way